sklearn/ensemble/tests/test_gradient_boosting_loss_functions.py

```diff
@@ -294,7 +294,7 @@ def test_init_raw_predictions_values():
             assert_allclose(raw_predictions[:, k], np.log(p))
 
 
-@pytest.mark.parametrize("seed", range(5))
+@pytest.mark.parametrize("seed", range(100))
 @pytest.mark.parametrize("alpha", [0.4, 0.5, 0.6])
 def test_lad_equals_quantiles(seed, alpha):
     # Make sure quantile loss with alpha = .5 is equivalent to LAD
```

Comment:
Please use the `global_random_seed` fixture here as well instead of exhaustive testing at each run.

```suggestion
@pytest.mark.parametrize("alpha", [0.4, 0.5, 0.6])
def test_lad_equals_quantiles(global_random_seed, alpha):
    # Make sure quantile loss with alpha = .5 is equivalent to LAD
```

The line:

```python
        rng = np.random.RandomState(seed)
```

needs to be updated but I cannot do it in the github web interface because it's not close enough to the diff.

Reply:
Thanks!